src/main/java/com/bookclub/controller/BooksController.java

```
@@ -102,16 +125,20 @@ protected void updateItem(Book book, boolean empty) {
     private void handleAddBook() {
         String title = titleField.getText();
         String author = authorField.getText();
-//        String genre = genreField.getText();
-        if (!title.isEmpty() && !author.isEmpty()) {
-            Book book = new Book(title, author);
+        String genre = genreComboBox.getValue();
+
+        if (!title.isEmpty() && !author.isEmpty() && genre !=null) {
+            Book book = new Book(title, author, genre);
             bookAO.addBook(book);
             updateBooks();
             titleField.clear();
             authorField.clear();
+            genreComboBox.setValue(null);
```

Is defaulting to an invalid value the normal use case/smart?

---

Done